server/consumer.go

```diff
@@ -58,7 +60,8 @@ type ConsumerInfo struct {
 	Paused         bool            `json:"paused,omitempty"`
 	PauseRemaining time.Duration   `json:"pause_remaining,omitempty"`
 	// TimeStamp indicates when the info was gathered
-	TimeStamp time.Time `json:"ts"`
+	TimeStamp time.Time         `json:"ts"`
+	PinnedIds map[string]string `json:"pinned_ids,omitempty"`
```

Comment:
Do we really want to include these? What if there are a very large number of these?

Reply:
It will come into play when we will start utilising Pirority Groups in numbers larger than 1. I assume we will not have large number of those, and for sure we will set some limits how many groups you can have per consumer.

@ripienaar do you agree?

Reply:
They will not be large, the background is that the combination of pinning and groups will make up partitioned consumption.  The number of groups will be limited to low numbers (tbd what that is though, today its 1)